content/en/docs/concepts/storage/volumes.md

```diff
@@ -1160,6 +1160,33 @@ spec:
 
 More examples can be found [here](https://github.com/kubernetes/examples/tree/master/staging/volumes/vsphere).
 
+#### CSI Migration
+
+{{< feature-state for_k8s_version="v1.19" state="beta" >}}
+
+The CSI Migration feature for vsphereVolume, when enabled, shims all plugin operations
+from the existing in-tree plugin to the `csi.vsphere.vmware.com` {{< glossary_tooltip text="CSI" term_id="csi" >}} Driver. In order to use this feature, the [vSphere CSI
+Driver](https://github.com/kubernetes-sigs/vsphere-csi-driver)
+must be installed on the cluster and the `CSIMigration` and `CSIMigrationvSphere`
+features must be enabled. This also requires minimum vSphere vCenter/ESXi Version to be 7.0u1 and minimum HW Version to be VM version 15.
+
+The following StorageClass parameters from vsphereVolume plugin will not be supported by vSphere CSI driver:
+
+* diskformat
+* hostfailurestotolerate
+* forceprovisioning
+* cachereservation
+* diskstripes
+* objectspacereservation
+* iopslimit
+
+Existing volumes created using these parameters will be migrated to the vSphere CSI driver, but new volumes created by the vSphere CSI driver will not be honoring these parameters.
```

Comment:
nit: I'd make this into a note shortcode

_beware_ that this might not preview correctly until commit 847f57ecf53b46bc7ab5e134b1e78ee2113fce21 is merged into [_dev-1.19_](https://github.com/kubernetes/website/tree/dev-1.19). You could cherry-pick that locally to verify that it's looking how you intend, then omit it from the branch you push.

```suggestion
{{< note >}}
The following StorageClass parameters from the built-in vsphereVolume plugin are not supported by the vSphere CSI driver:

* `diskformat`
* `hostfailurestotolerate`
* `forceprovisioning`
* `cachereservation`
* `diskstripes`
* `objectspacereservation`
* `iopslimit`

Existing volumes created using these parameters will be migrated to the vSphere CSI driver, but new volumes created by the vSphere CSI driver will not be honoring these parameters.
{{< /note >}}
```

Reply:
Looks like commit 847f57e is merged into master.  My PR will be merged into dev-1.19.  Probably should make this a separate PR when that commit is backported to dev-1.19?

Reply:
_master_ will be merged into _dev-1.19_ to incorporate that kind of change (a regular thing). It's fine to do this as a separate PR after that happens.

(It's also fine IMO to merge in a note shortcode that looks OK now but will render even better with changes from 847f57ecf53b46bc7ab5e134b1e78ee2113fce21)

Reply:
Sure.  I added it as a note.